docs/docs-ref-autogen/excel/excel/excel.range.yml

```diff
@@ -1328,8 +1328,11 @@ methods:
       Finds the given string based on the criteria specified. If the current
       range is larger than a single cell, then the search will be limited to
       that range, else the search will cover the entire sheet starting after
-      that cell. If there are no matches, this function will return a null
-      object.
+      that cell. If there are no matches, then this function will return an
+      object with its `isNullObject` property set to `true`<!-- -->. For further
```

Comment:
The isNullObject comments all have `<!-- -->` included. Should we be concerned about this?

Reply:
@alison-mk For things like this, you can verify the behavior by checking the review site for this branch. Let me know if you need the link

Reply:
Thanks! The review site output looks fine to me.